clusterloader2/pkg/measurement/common/wait_for_pods.go

```diff
@@ -74,37 +75,89 @@ func (*waitForRunningPodsMeasurement) Execute(config *measurement.MeasurementCon
 	time.AfterFunc(timeout, func() {
 		close(stopCh)
 	})
-	return summaries, waitForPods(config.ClientSet, namespace, labelSelector, fieldSelector, desiredPodCount, stopCh, true)
+	return summaries, waitForPods(config.ClientSet, namespace, labelSelector, fieldSelector, desiredPodCount, stopCh, true, waitForRunningPodsMeasurementName)
 }
 
 // Dispose cleans up after the measurement.
 func (*waitForRunningPodsMeasurement) Dispose() {}
 
-func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldSelector string, desiredPodCount int, stopCh <-chan struct{}, log bool) error {
+type podsStartupStatus struct {
+	Expected           int
+	Terminating        int
+	Running            int
+	RunningButNotReady int
+	Waiting            int
+	Pending            int
+	Unknown            int
+	Inactive           int
+	Created            int
+}
+
+// String returns string representation for podsStartupStatus.
+func (s *podsStartupStatus) String() string {
+	return fmt.Sprintf("Pods: %d out of %d created, %d running, %d not scheduled, %d pending scheduled, %d inactive, %d terminating, %d unknown, %d runningButNotReady ",
+		s.Created, s.Expected, s.Running, s.Pending, s.Waiting, s.Inactive, s.Terminating, s.Unknown, s.RunningButNotReady)
```

Comment:
Now it's incorrect order - not scheduled are Waiting, pendning scheduled are Pending.

Reply:
Order fixed.